turbinia/e2e/googleclouddisk.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+# This Turbinia e2e test will test the googleclouddisk functionality.
+# It will try to process a disk created from the image 'test-disk2'.
+
+
+TURBINIA_CLI="turbiniactl"
+MAIN_LOG="main.log"
+STATS_LOG="stats.log"
+DETAIL_LOG="reqdetails.log"
+OUT_TGZ="reqresults.tgz"
+DISK="test-disk2"
+
+if [ $# -ne  2 ]
+then
+  echo "Not enough arguments supplied, please provide project and zone."
+  echo "$0 [PROJECT] [ZONE]"
+  exit 1
+fi
+
+PROJECT="$1"
+ZONE="$2"
+
+echo -n "Started at "
+date -Iseconds
+
+echo "Creating unique request ID...."
+REQ_ID=`uuidgen -rt`
+
+echo "Creating GCE test disk to use in e2e test"
+gcloud --project=$PROJECT compute disks create $DISK --image=$DISK --zone=$ZONE
+
+echo "Executing googlecloudisk e2e test....this takes ~60 minutes!"
+$TURBINIA_CLI -r $REQ_ID -L $MAIN_LOG -a -w googleclouddisk -d $DISK -z $ZONE
+
+# When the Turbinia request is finished request the final request statistics.
```

Comment:
Just a question to @aarontp - Lets support a machine readable output here as well so we don't have to do this log line parsing?

Reply:
Aaron has created JSON output for the CLI with below PR on my request just for that purpose :). Will rewrite this to parse the JSON in a second iteration, for now I would like the e2e stuff in there so we can write more e2e tests.
https://github.com/google/turbinia/pull/582

Reply:
Yeah, doing this in a second iteration sounds good.